### DISCUSSION TITLE
A random doubt regarding using fineutned model

### DISCUSSION THREAD

**Discussion by chiragbrudite** [axolotl-ai-cloud/axolotl#1564] [Q&A]
I am new to industry i have to finetune LLM mixtral MOE, i have not used axololt till now but I want to know is it possible to finetune the model here and use it(weights) in the software I am developing. 
I want to use axololt with any gpu platform to finetune the model and use the saved model in my application or deploy on my cloud and utilise the finetuned model to get responses from aaplication. 

thanks

**Comment by NanoCode012** [comment]
Late update, for any future readers, yes. The models saved are transformers compatible which works across the ecosystem.
